affectedmoduledetector/src/test/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetectorImplTest.kt

```diff
@@ -104,29 +109,29 @@ class AffectedModuleDetectorImplTest {
             .withParent(root)
             .build()
         p3 = ProjectBuilder.builder()
-            .withProjectDir(tmpDir.resolve("p1:p3"))
+            .withProjectDir(tmpDir.resolve("p1/p3"))
             .withName("p3")
             .withParent(p1)
             .build()
         val p3config = p3.configurations.create("p3config")
         p3config.dependencies.add(p3.dependencies.project(mutableMapOf("path" to ":p1")))
         p4 = ProjectBuilder.builder()
-            .withProjectDir(tmpDir.resolve("p1:p3:p4"))
```

Comment:
It's really convenient to specify the directory like this as it looks like a representation of the Gradle projects (I assume) but `:` is an illegal character in the Windows file system so this breaks the test on Windows.